guides/plugins/apps/administration/flow-builder/add-custom-flow-actions-from-app-system.md

```diff
@@ -0,0 +1,255 @@
+    <meta>
+        <name>FlowBuilderActionApp</name>
+        <label>Flow Builder Action App</label>
+        <label lang="de-DE">Flow Builder Action App DE</label>
+        <description>This is the example description for app</description>
+        <description lang="de-DE">This is the example description for app</description>
+        <author>shopware AG</author>
+        <copyright>(c) shopware AG</copyright>
+        <version>4.14.0</version>
+        <icon>Resources/app.png</icon>
+        <license>MIT</license>
+    </meta>
+</manifest>
+```
+{% endcode %}
+
+{% hint style="warning" %}
+The name of your app that you provide in the manifest file needs to match the folder name of your app.
+{% endhint %}
+
+## Define the flow action
+
+To create a flow action, you need to define a `<flow-action>` block within a file called `flow-action.xml`. Each `<flow-action>` represents one action and you can define an arbitrary number of actions.
+
+{% code title="Resources/flow-action.xml" %}
+```xml
+<flow-actions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/FlowAction/Schema/flow-action-1.0.xsd">
+    <flow-action>
+        ... # The first action
+    </flow-action>
+    <flow-action>
+        ... # The second action
+    </flow-action>
+    <flow-action>
+        ... # The third action
+    </flow-action>
+    ...
+</flow-actions>
+```
+{% endcode %}
+
+A single flow action would look like this:
+
+```xml
+<flow-action>
+    <meta>
+        <name>slack.send.message</name>
+        <label>Send slack message</label>
+        <label lang="de-DE">Send slack message</label>
+        <badge>Slack</badge>
+        <description>Slack send message description</description>
+        <description lang="de-DE">Slack send message description DE</description>
+        <url>https://hooks.slack.com/services/{id}</url>
+        <sw-icon>default-communication-speech-bubbles</sw-icon>
+        <icon>slack.png</icon>
+        <requirements>orderAware</requirements>
+        <requirements>customerAware</requirements>
+    </meta>
+    ...
+</flow-action>
+```
+
+| Key | Required | Description |
+| :--- | :--- | :--- |
+| name | yes | The technical name of your action, unique for all actions |
+| label | yes | A name to be shown for your action in the actions list or action modal title |
+| badge | no | An attached badge shown behind the label in the action modal title |
+| description | yes | Detailed information for your action |
+| sw-icon | no | An icon component name from the [icon library](https://component-library.shopware.com/icons/) |
+| icon | no | Alternatively, a path to your action icon |
+| requirements | yes | Available action triggers, read more below |
```

Comment:
This is a little bit unclear for me, the requirements specify additional data that your action needs and according to the requirements a subset of action triggers can be used that fulfill the requirements